platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/telemetry/PerformanceEventTest.java

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.mapboxsdk.testapp.telemetry;
+
+import android.os.Bundle;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@RunWith(AndroidJUnit4.class)
+public class PerformanceEventTest {
+
+    @Test
+    public void triggerPerformanceEvent() {
+      String now =
+        new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date());
+      Bundle bundle = new Bundle();
+      bundle.putBoolean("test", true);
+      bundle.putString("date", now);
+
+      Mapbox.getTelemetry().onPerformanceEvent(bundle);
```

Comment:
after verifying the changes made to `https://github.com/mapbox/mapbox-gl-native/pull/13894/files#diff-22a9d97e0bf7f0f0d197b82c8f2345c2R17`, I'm not seeing how above will be correctly ingested as it doesn't follow that definition completely. I will PM you the definition